recipe/meta.yaml

```diff
@@ -1,28 +1,40 @@
-{% set name = "tokenizers" %}
 {% set version = "0.13.1" %}
 
 package:
-  name: {{ name|lower }}
+  name: tokenizers
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 3333d1cee5c8f47c96362ea0abc1f81c77c9b92c6c3d11cbf1d01985f0d5cf1d
+  url: https://github.com/huggingface/tokenizers/archive/refs/tags/python-v{{ version }}.tar.gz
+  sha256: 41cff8c8c87ba6dfbd9eb1d89b006aabb9c9823ffd09e281d6ddfb9ae695bd1a
+  patches:
+    - patches/0001-don-t-fork-on-windows.patch  # [win]
```

Comment:
Nit:

```suggestion
    - patches/0001-dont-fork-on-windows.patch  # [win]
```

Reply:
This is generated from the git commit messages with `git format-patch`, it makes no sense to police these IMO (though if you want I can change the commit message of the patch).

Reply:
Not policing it, just recommending what I think is cleaner. But that's fine.